### PR TITLE
Support jdk6

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: client/src/main/java/com/vesoft/nebula/graph/client/GraphClientImpl.java [hunk 1]
```diff
@@ -200,13 +198,12 @@ public ResultSet executeQuery(String statement) throws ConnectionException,
     }
 
     private boolean checkTransportOpened(TTransport transport) {
-        return !Objects.isNull(transport) && transport.isOpen();
+        return !(transport == null) && transport.isOpen();
```
Comment (jude-zhu) [inline client/src/main/java/com/vesoft/nebula/graph/client/GraphClientImpl.java]: `!(transport == null)` that is `transport != null`

File: client/src/main/java/com/vesoft/nebula/meta/client/MetaClientImpl.java [hunk 2]
```diff
@@ -334,8 +330,8 @@ private boolean getEdgeTypes(int spaceId) {
 
         if (response.getCode() == ErrorCode.SUCCEEDED) {
             List<EdgeItem> edgeItem = response.getEdges();
-            Map<String, EdgeItem> tmp = new HashMap<>();
-            if (!Objects.isNull(edgeItem)) {
+            Map<String, EdgeItem> tmp = Maps.newHashMap();
+            if (!(edgeItem == null)) {
```
Comment (jude-zhu) [inline client/src/main/java/com/vesoft/nebula/meta/client/MetaClientImpl.java]: ditto

File: examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java [hunk 3]
```diff
@@ -67,85 +66,88 @@ public static void main(String[] args) {
...
         }
 
+        GraphClient client = new GraphClientImpl(args[0], Integer.valueOf(args[1]));
         try {
-            try (GraphClient client = new GraphClientImpl(args[0], Integer.valueOf(args[1]))) {
-                client.connect("user", "password");
-                int code = client.switchSpace(SPACE_NAME);
+            client.connect("user", "password");
+            int code = client.switchSpace(SPACE_NAME);
+            if (code != ErrorCode.SUCCEEDED) {
```
Comment (jude-zhu) [inline examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java]: p.s. I prefer to put the constant before the variable in a comparison

**Comment by jude-zhu** [comment]
`java.lang.AutoCloseable` that `com.vesoft.nebula.Session` extends is since java 1.7+

File: client/src/main/java/com/vesoft/nebula/graph/client/GraphClientImpl.java (hunk 1 above)
Comment (darionyaphet) [reply client/src/main/java/com/vesoft/nebula/graph/client/GraphClientImpl.java]: Smart Catch ！

File: examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java (hunk 3 above)
Comment (darionyaphet) [reply examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java]: `ErrorCode.SUCCEEDED != code` is better ?

**Pull request merged**